### PR TITLE
Support ZGC

### PR DESCRIPTION
**Pull request by xu1009**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<8411>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
I think you submitted this PR too early? Protocol PR has been merged, so you don't have correct submodule.

File: oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/jvm/JVMSourceDispatcher.java [hunk 1]
```diff
@@ -106,6 +106,8 @@ private void sendToGCMetricProcess(String service,
                     break;
                 case OLD:
                     serviceInstanceJVMGC.setPhrase(GCPhrase.OLD);
```
Comment (wu-sheng) [inline oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/jvm/JVMSourceDispatcher.java:108]: Miss break.

Comment (xu1009) [reply oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/jvm/JVMSourceDispatcher.java:108]: ok,fix it

**Comment by xu1009** [comment]
> I think you submitted this PR too early? Protocol PR has been merged, so you don't have correct submodule.

ok, the proto pr  https://github.com/apache/skywalking-data-collect-protocol/pull/59

**Comment by wu-sheng** [comment]
I have updated the submodule, please resolve the conflicts, thanks.

**Comment by xu1009** [comment]
ok

**Comment by wu-sheng** [comment]
Please fix the CI.

**Comment by xu1009** [comment]
the failed  ci check seems no relation with this commit?

**Comment by wu-sheng** [comment]
The error is strange. Could you try to close this PR and open another one? I feel this is CI env issue.

**Comment by xu1009** [comment]
ok

**Pull request closed without merging**